src/TestLogger/TestLogger.csproj

```diff
@@ -9,13 +9,29 @@
     <TargetFramework>netstandard1.5</TargetFramework>
     <AssemblyName>Spekt.TestLogger</AssemblyName>
     <Version>$(PackageVersion)</Version>
-    <IsPackable>false</IsPackable>
+    <IsPackable>true</IsPackable>
   </PropertyGroup>
 
   <PropertyGroup Label="Configuration">
     <RootNamespace>Spekt.TestLogger</RootNamespace>
   </PropertyGroup>
 
+  <!-- Nuget properties. Testlogger will be a local package for internal spekt consumption. -->
+  <PropertyGroup>
+    <Authors>Spekt contributors</Authors>
+    <Description>Core test logger for Spekt</Description>
```

Comment:
@codito I'm just thinking about clarity when look at this on nuget. I have seen packages like his which had a warning of sorts saying not to take a dependency on the package directly, but to choose some other package. To that end, we might also want to call this TestLogger.Core.

Reply:
I'm thinking we will not publish this on nuget. It will be internal only for spekt packages to consume (e.g. junit.testlogger or nunit.testlogger), and those packages will `privately` package this dll in their nuget package. This will help achieve our primary goal of reusable core abstractions.

The problem with `TestLogger` in the dll name is vstest.console and `dotnet test` will try to reflect through it to find loggers. We'll probably rename it to something like CoreLogger. How does it sound?

Reply:
Oh, I see. That solution didn't occur to me, but a private package makes sense. CoreLogger works for me.